Sources/WebSocketKit/WebSocket.swift

```diff
@@ -51,6 +62,11 @@ public final class WebSocket {
         self.onTextCallback = callback
     }
 
+    /// The same as `onText`, but with raw data instead of the decoded `String`.
```

Comment:
What purpose does this callback serve? I don't see any usage of it anywhere, including in the tests.

Reply:
Currently we only have `onTextCallback`. What is sent to the ws is, of course, Data, not string. but when using `onTextCallback`, ws-kit turns the data into a string and passes the string to the users of the package. The problem is that if the text is for example in JSON format, ws-kit users need to turn the string into Data again to pass it to somewhere like JSONDecoder. so we have `Data -> String -> Data` instead of just `Data` which is wasteful. this new callback solves that problem.

Reply:
like it is mentioned in this issue: https://github.com/vapor/websocket-kit/issues/79

Reply:
i should add some tests, never-the-less.

Reply:
add a test to assert both text callbacks have the same behavior.
I should also add that `onBinary` is not the same as `onTextBuffer` because `onBinary` only is activated if the ws frame is an actual binary frame. `onTextBuffer` is for when the ws frame is a text frame, but users might still prefer to access the string's data directly. I did try to mix the two, but is could cause problems.